core/templates/components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts

```diff
@@ -114,6 +115,7 @@ export class CkEditorInitializerService {
                 rteHelperService.openCustomizationModal(
                   customizationArgSpecs,
                   customizationArgs,
+                  componentId,
```

Comment:
Make this the first argument (and fix all callsites) since the ID is more "primary" than the customization args.

Reply:
done